### DISCUSSION TITLE
How do I save heat maps in the flow?

### DISCUSSION THREAD

**Discussion by huang-kaixin** [The-OpenROAD-Project/OpenROAD-flow-scripts#1535] [Q&A]
![heat maps](https://github.com/The-OpenROAD-Project/OpenROAD-flow-scripts/assets/111551571/0de37bb6-4780-4391-a738-87332bc3a4cd)

**Comment by rovinski** [comment]
If you would like to save the image, I believe you can just use `save_image`.
If you would like to save the raw data, it can be saved in CSV format using `gui::dump_heatmap`
See more info in the [GUI docs](https://openroad.readthedocs.io/en/latest/main/src/gui/README.html)
